### PR TITLE
fix: add Catalog when discover ff is false

### PR DESCRIPTION
**Pull request by RamyEB**
### 📝 Description

Fix Discover crash on android

### ❓ Context

- **Impacted projects**: `LLM` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/RootNavigator/DiscoverNavigator.tsx [hunk 1]
```diff
@@ -20,21 +20,32 @@ export default function DiscoverNavigator() {
...
+          />
+          <Stack.Screen
+            name={ScreenName.PlatformCatalog}
+            component={Catalog}
+            options={{
+              headerShown: false,
+            }}
+          />
+        </>
       )}
```
Comment (JunichiSugiura) [inline apps/ledger-live-mobile/src/components/RootNavigator/DiscoverNavigator.tsx:49]: Simply as this?
```javascript
<Stack.Navigator screenOptions={stackNavigationConfig}>
  {config?.enabled && config?.params.version === "2" && (
    <Stack.Screen
      name={ScreenName.DiscoverScreen}
      ..
    />
  )}
  <Stack.Screen
    name={ScreenName.PlatformCatalog}
    ..
  />
</Stack.Navigator>
```

**Review by JunichiSugiura — CHANGES_REQUESTED** [review]

File: apps/ledger-live-mobile/src/components/RootNavigator/DiscoverNavigator.tsx (hunk 1 above)
Comment (RamyEB) [reply apps/ledger-live-mobile/src/components/RootNavigator/DiscoverNavigator.tsx:49]: This is what was originally coded, but ios or android have different behaviours and don't work as expected
With this ios still open the Menu

**Review by alexandremgo — APPROVED** [review]
Tested on Android, looks good ;)

**Review by JunichiSugiura — APPROVED** [review]

**Comment by JunichiSugiura** [comment]
🙆‍♂️

https://user-images.githubusercontent.com/8398372/236454446-e58b8e2d-bc62-48b0-9301-57fb398c0bb8.mov


https://user-images.githubusercontent.com/8398372/236454453-b5691e16-c948-4e6a-bc39-f53389e47468.mov

**Pull request merged**